recipes/opentelemac/meta.yaml

```diff
@@ -85,7 +85,7 @@ outputs:
         - matplotlib-base
         - mpi4py
         - numpy >=1.25
-        - python >=3.9
+        - python >=3.9,<3.12
```

Comment:
why wouldn't we include 3.12?

Reply:
https://gitlab.pam-retd.fr/otm/telemac-mascaret/-/issues/946

Reply:
ok, I don't think this is a priority on the opentelemac roadmap atm..